src/main/java/net/geforcemods/securitycraft/SCContent.java

```diff
@@ -223,11 +228,13 @@
 	@RegisterItemBlock(SCItemGroup.EXPLOSIVES) public static final RegistryObject<Block> GOLD_ORE_MINE = BLOCKS.register("gold_mine", () -> new BaseFullMineBlock(Material.ROCK, SoundType.STONE, Blocks.GOLD_ORE, 3.0F));
 	@RegisterItemBlock(SCItemGroup.EXPLOSIVES) public static final RegistryObject<Block> IRON_ORE_MINE = BLOCKS.register("iron_mine", () -> new BaseFullMineBlock(Material.ROCK, SoundType.STONE, Blocks.IRON_ORE, 3.0F));
 	@RegisterItemBlock(SCItemGroup.EXPLOSIVES) public static final RegistryObject<Block> COAL_ORE_MINE = BLOCKS.register("coal_mine", () -> new BaseFullMineBlock(Material.ROCK, SoundType.STONE, Blocks.COAL_ORE, 3.0F));
+	@RegisterItemBlock(SCItemGroup.EXPLOSIVES) public static final RegistryObject<Block> NETHER_GOLD_ORE_MINE = BLOCKS.register("nether_gold_mine", () -> new BaseFullMineBlock(Material.ROCK, SoundType.NETHER_GOLD, Blocks.NETHER_GOLD_ORE, 3.0F));
 	@RegisterItemBlock(SCItemGroup.EXPLOSIVES) public static final RegistryObject<Block> LAPIS_ORE_MINE = BLOCKS.register("lapis_mine", () -> new BaseFullMineBlock(Material.ROCK, SoundType.STONE, Blocks.LAPIS_ORE, 3.0F));
 	@RegisterItemBlock(SCItemGroup.EXPLOSIVES) public static final RegistryObject<Block> DIAMOND_ORE_MINE = BLOCKS.register("diamond_mine", () -> new BaseFullMineBlock(Material.ROCK, SoundType.STONE, Blocks.DIAMOND_ORE, 3.0F));
 	@RegisterItemBlock(SCItemGroup.EXPLOSIVES) public static final RegistryObject<Block> REDSTONE_ORE_MINE = BLOCKS.register("redstone_mine", () -> new RedstoneOreMineBlock());
 	@RegisterItemBlock(SCItemGroup.EXPLOSIVES) public static final RegistryObject<Block> EMERALD_ORE_MINE = BLOCKS.register("emerald_mine", () -> new BaseFullMineBlock(Material.ROCK, SoundType.STONE, Blocks.EMERALD_ORE, 3.0F));
-	@RegisterItemBlock(SCItemGroup.EXPLOSIVES) public static final RegistryObject<Block> QUARTZ_ORE_MINE = BLOCKS.register("quartz_mine", () -> new BaseFullMineBlock(Material.ROCK, SoundType.STONE, Blocks.NETHER_QUARTZ_ORE, 3.0F));
+	@RegisterItemBlock(SCItemGroup.EXPLOSIVES) public static final RegistryObject<Block> QUARTZ_ORE_MINE = BLOCKS.register("quartz_mine", () -> new BaseFullMineBlock(Material.ROCK, SoundType.NETHER_ORE, Blocks.NETHER_QUARTZ_ORE, 3.0F));
+	@RegisterItemBlock(SCItemGroup.EXPLOSIVES) public static final RegistryObject<Block> GILDED_BLACKSTONE_MINE = BLOCKS.register("gilded_blackstone_mine", () -> new BaseFullMineBlock(Material.ROCK, SoundType.GILDED_BLACKSTONE, Blocks.GILDED_BLACKSTONE, 1.5F));
```

Comment:
I don't think it makes sense to add an _ore_ mine for something that isn't an ore and is only present in  bastion remnants. It would make more sense to add an ore mine for ancient debris, as it actually spawns in the world like ores.

Reply:
Ancient Debris Mine is a very good idea, but I'm unsure about removing the Gilded Blackstone Mine. It is technically an ore, you can mine it to get gold (in some cases). If a player discoveres a bastion and replaces some of the Gilded Blackstone there with mines, that'd be a perfect trap.

Reply:
Fair enough.